data/human/pirate jobs.txt

```diff
@@ -3396,3 +3396,21 @@ mission "Republic Assassination [4]"
 		"reputation: Republic" -= 25
 		"reputation: Navy (Oathkeeper)" -= 25
 		"reputation: Pirate" += 10
+
+mission "Slave revolt"
+	landing
+	repeat
+	to offer
+		random < 10
+	on offer
+		require "Slave" 2
```

Comment:
This is pretty often, 10% or 1 every ten landing for 100% instant death, even if it's to suggest Slave owning is bad there's just better way to do this and piss people off with uncontrollable instant death, we got enough of that just flying around in space. 
Especially for just two slaves you somehow didn't figure out how to keep them properly or lock your cabin door.